HIRS_Utils/src/main/java/hirs/data/persist/certificate/PlatformCredential.java

```diff
@@ -73,6 +72,13 @@
     private static final String TCG_CREDENTIAL_SPECIFICATION = "2.23.133.2.23";
     private static final String PLATFORM_CONFIGURATION_URI = "2.23.133.5.1.3";
     private static final String PLATFORM_CONFIGURATION = "2.23.133.5.1.7.1";
+    private static final String PLATFORM_CONFIGURATION_V2 = "2.23.133.5.1.7.2";
+
+    //TCG Platform Specification values
```

Comment:
Can we get a reference to where in the specification these values come from, as part of this comment.

Reply:
These are currently placeholders, we are waiting on the correct information.  I'll add a comment about that.